File: src/main/java/me/nasukhov/TukitaLearner/bot/io/Output.java
package me.nasukhov.TukitaLearner.bot.io;

import java.util.Map;

public interface Output {
    void write(String text);
    void promptChoice(String question, Map<String, String> replyOptions);

    default String mention(String userId) {
        return String.format("<user>%s</user>", userId);
    }
}

File: src/main/kotlin/me/nasukhov/TukitaLearner/bot/io/Output.kt
package me.nasukhov.TukitaLearner.bot.io

interface Output {
    fun write(text: String)
    fun promptChoice(question: String, replyOptions: Map<String, String>)

    fun mention(userId: String): String {
        return "<user>$userId</user>"
    }
}
